website/frontend/src/consts.ts

```diff
@@ -5,6 +5,8 @@ export const isProd = process.env.NODE_ENV === 'production';
 export const JSONRPC_VERSION = '2.0';
 export const INITIAL_LIMIT_BY_PAGE = 13;
 export const GEAR_BALANCE_TRANSFER_VALUE = 10_000_000_000;
+export const WASM_COMPILER_BUILD = 'https://idea.gear-tech.io/wasm-compiler/build';
```

Comment:
https://idea.gear-tech.io/ has to be in environment variables, beacuse it could change